### PR TITLE
Bump Prio2 to VDAF-13 Take #2

### PR DESCRIPTION
**Pull request by jhoyla**

### CODE REVIEW

**Review by mendess — APPROVED** [review]

File: crates/daphne/src/vdaf/prio2.rs [hunk 1]
```diff
@@ -71,7 +71,8 @@ pub(crate) fn prio2_prep_init(
     <()>::get_decoded_with_param(&vdaf, public_share_data)?;
     let input_share: Share<FieldPrio2, 32> =
         Share::get_decoded_with_param(&(&vdaf, agg_id), input_share_data)?;
```
Comment (cjpatton) [inline crates/daphne/src/vdaf/prio2.rs:73]: I'd add a comment here nothing that the `ctx` string is ignored by Prio2.

File: crates/daphne/src/vdaf/prio2.rs [hunk 2]
```diff
@@ -133,7 +134,7 @@ pub(crate) fn prio2_prep_finish(
             )))
         }
     };
-    let agg_share = VdafAggregateShare::Field32Draft09(vdaf.aggregate(&(), [out_share])?);
```
Comment (cjpatton) [inline crates/daphne/src/vdaf/prio2.rs:136]: We should be able to remove the `Field32Draft09` variant now.

**Review by cjpatton — CHANGES_REQUESTED** [review]
Whoops, I meant to request changes, not just comment.

File: crates/daphne/src/vdaf/prio2.rs (hunk 2 above)
Comment (jhoyla) [reply crates/daphne/src/vdaf/prio2.rs:136]: Doesn't `Pine32HmacSha256Aes128` depend on it?

**Pull request merged**